### ISSUE TITLE
is it possible to not invert the color of images?

### ISSUE THREAD

**Issue by kurokirasama** [PalmerAL/min-dark-mode-userscript#3]
great tool btw!!

**Comment by PalmerAL** [comment]
Do you have an example URL? Actual `<img>` elements should remain non-inverted, but other elements with a background-image will be, which is harder to fix.

**Comment by kurokirasama** [comment]
habitica.com... maybe they have background images...

**Comment by PalmerAL** [comment]
Yup, that's a background image. If you use that site a lot, you can exclude specific elements by editing the last CSS rule:

```
  iframe, img, video, canvas, #intro-signup {
  filter: hue-rotate(180deg) invert(100%) !important;
  }
```

Or disable the script for that site using these options: https://github.com/minbrowser/min/wiki/userscripts

But I can't think of an easy way to fix this for all sites at the moment.

**Comment by kurokirasama** [comment]
thanks!! HOw do I discover what elemente to exclude? For instance, how do I exclude the <head> of the page?

**Comment by PalmerAL** [comment]
If you wanted to exclude specific elements, you'd need to use the "elements" panel of the devtools (developer > inspect page), and look for IDs or classnames of the elements once you find them.

Particularly if the website has a dark theme, it's probably easier to exclude the entire site using the instructions linked above.

**Comment by kurokirasama** [comment]
thanks!!! Sorry to bother you again, but what do I exactly add to the script to exclude, for instance the <head>...</head> part of the page...

**Comment by PalmerAL** [comment]
The last rule in the script (on [this line](https://github.com/PalmerAL/min-dark-mode-userscript/blob/master/dark-mode.js#L21)) makes elements be excluded, so once you have a CSS selector for the element, you would add it to that list (separated by a comma).
